collect_app/build.gradle

```diff
@@ -236,6 +236,7 @@ dependencies {
     implementation "com.google.zxing:core:3.3.0"
     implementation "com.journeyapps:zxing-android-embedded:3.5.0"
     implementation "net.danlew:android.joda:2.9.9"
+    implementation 'com.github.chanmratekoko:myanmar-calendar:1.0.5.RELEASE'
```

Comment:
Small optimization but would be nice to put the bikram sambat import next to this one and add a comment making it clear they're used for specialized calendars. Would also be good to match the quote type used in all other imports (" rather than ').